.github/workflows/ci.yml

```diff
@@ -0,0 +1,146 @@
+---
+name: CI
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+  pull_request:
+
+permissions:
+  contents: read
+  packages: write
+  checks: write
+  pull-requests: write
+
+jobs:
+  test:
+    if: false # TODO: make tests faster and enable this job
```

Comment:
Tests require too much time. It makes sense at first make them faster and then enable this job.